.github/workflows/auto-add-issues.yml

```diff
@@ -0,0 +1,29 @@
+name: Label new issues as needs-triage and add to CodeFlare Sprint Board
+on:
+  issues:
+    types:
+      - opened
+jobs:
+  add_label:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.issues.addLabels({
```

Comment:
Thinking whether it would have more sense to leverage GitHub CLI here.

Reply:
Updated.